crates/typst/src/visualize/polygon.rs

```diff
@@ -72,6 +74,11 @@ impl PolygonElem {
         #[named]
         fill: Option<Option<Paint>>,
 
+        /// The rule used to fill the polygon.
+        #[named]
+        #[default(FillRule::default())]
+        fill_rule: FillRule,
```

Comment:
Does the option even make sense here? Regular polygons are always "odd" everywhere so the two fill rule options would behave the same.